src/main/java/frc/robot/configs/PracticeRobotConfig.java

```diff
@@ -0,0 +1,394 @@
+package frc.robot.configs;
+
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.util.Units;
+import frc.lib.team3061.RobotConfig;
+import frc.lib.team3061.drivetrain.swerve.SwerveConstants.SwerveType;
+
+/*
+ * Refer to the README for how to represent your robot's configuration. For more information on
+ * these methods, refer to the documentation in the RobotConfig class.
+ */
+public class PracticeRobotConfig extends RobotConfig {
```

Comment:
What is this class for? Should it be deleted? There's already another config file for the practice bot.